suite-common/wallet-types/src/selectedAccount.ts

```diff
@@ -0,0 +1,77 @@
+import { Network } from '@suite-common/wallet-config';
+
+import { Discovery } from './discovery';
+import { Account, WalletParams } from './account';
+
+// Context notifications view
+// Account is in "watch only" mode
+export type AccountWatchOnlyMode =
```

Comment:
All these types in this file should be prefixed with `Selected` otherwise it's very confusing with normal account types